frontend/src/views/MyProgress/index.vue

```diff
@@ -221,6 +224,7 @@ export default {
       centralKitchenDiagnosticMode: null,
       cancelDialog: false,
       campaignEndDate: window.TELEDECLARATION_END_DATE ? new Date(window.TELEDECLARATION_END_DATE) : null,
+      thisYear: lastYear() + 1,
```

Comment:
Elsewhere in the app we use `currentYear` which is probs clearer since it doesn't bring the `this` instance notion
```suggestion
      currentYear: lastYear() + 1,
```